idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -0,0 +1,411 @@
+    CONFIG.thermodynamic_assumption = ThermodynamicAssumption.isentropic
+    CONFIG.get("thermodynamic_assumption")._default = \
+        ThermodynamicAssumption.isentropic
+    CONFIG.get('thermodynamic_assumption')._domain = \
+        In([ThermodynamicAssumption.isentropic])
+
+    ################################################
+
+    def build(self):
+        super(InletStageData, self).build()
+        config = self.config # shorter config pointer
+
+        # The thermodynamic expression writer object, te, writes expressions
+        # including external function calls to calculate thermodynamic quantities
+        # from a set of state variables.
+        te = ThermoExpr(blk=self, parameters=config.property_package)
+
+        if SolverFactory('ipopt').available():
+          solver = SolverFactory('ipopt')
+          solver.options = {'tol':1e-6,
+                            'max_iter': 100,
+                            'linear_solver': 'ma27'}
+        else:
+          solver = None
+
+        #####################################################
+        # Declare variables for the model
+
+        # Some shorter refernces to property blocks
+        properties_in = self.control_volume.properties_in
+        properties_out = self.control_volume.properties_out
+
+        
+        self.kappaT = Var(self.flowsheet().config.time,
+                            initialize=5.0e-7)
+        self.c0 = Var(self.flowsheet().config.time,
+                        initialize = 260,
+                        doc="Speed of sound in the inlet gas stream",
+                        bounds=(0,4000)) # m/s
+        self.mass_flow_coeff = Var(self.flowsheet().config.time,
+                            initialize=0.0735,
+                            doc="Compressor flow coefficient",
+                            bounds=(0.01,0.15))        
+        self.I = Var(self.flowsheet().config.time,
+                            initialize=0.6993,
+                            doc="Impeller work coefficient",
+                            bounds=(0.001,1))
+        self.mu_p_v = Var(self.flowsheet().config.time,
+                            initialize = 0.60,
+                            doc="Polytropic head coefficient vaned diffuser",
+                            bounds=(0.001,1))        
+        self.mu_p = Var(self.flowsheet().config.time,
+                            initialize = 0.60,
+                            doc="Polytropic head coefficient",
+                            bounds=(0.001,1))
+        self.eff_p_v = Var(self.flowsheet().config.time,
+                            initialize=0.85,
+                            doc="Polytropic stage efficiency vaned diffuser",
+                            bounds=(0.001,1))
+        self.eff_p = Var(self.flowsheet().config.time,
+                            initialize=0.85,
+                            doc="Polytropic stage efficiency",
+                            bounds = (0.001, 1.0))       
```

Comment:
Check PEP8 formatting. Indentation is inconsistent, and there should not be spaces around the `=` here.

Reply:
It is revised.